Library/Formula/gdk-pixbuf.rb

```diff
@@ -21,6 +21,9 @@ class GdkPixbuf < Formula
   depends_on "libpng"
   depends_on "gobject-introspection"
 
+  # disable the default relocatable library setting
+  patch :DATA
```

Comment:
I am going to report this to the developer as I am starting to consider it as a bug: he makes the following assumption in configure.ac:

````
Enable replacing the build-time prefix in the loaders
cache with the installation prefix on this machine
for relocatable packages such as Windows and OS X
applications and linux bundles
````

Now IMHO this should certainly not be the default behavior on Mac OS X as most people that will install gdk-pixbuf will do it via Homebrew or MacPorts where this is certainly not desirable. The problem is that the developer did not turn this into an option, but instead hardcoded this :-(